app/experimenter/experiments/models/nimbus.py

```diff
@@ -177,7 +184,7 @@ def treatment_branches(self):
     @property
     def start_date(self):
         start_changelog = self.changes.filter(
-            old_status=self.Status.ACCEPTED, new_status=self.Status.LIVE
+            old_status=self.Status.DRAFT, new_status=self.Status.LIVE
```

Comment:
I think we need to include DRAFT -> LIVE and PREVIEW -> LIVE here, since a launch can be requested from PREVIEW status also.

Reply:
Ah nice catch!